README.md

```diff
@@ -111,6 +111,9 @@ hierarchy.  These are not really internal to the library.  We classify
   version number.  As yet unreleased modules and APIs reside in the
   `Internal` namespace.
 
+You can use these Internal modules in your code, but if you do so please use a
+minor release upper bound to adhere to the Haskell PVP.
```

Comment:
Please use a minor release upper bound to adhere to the Haskell PVP when using these Internal modules.